File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ShippingQueryFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.payments.ShippingQuery;

public class ShippingQueryFilterValidator extends FilterValidator<ShippingQuery> {

    public ShippingQueryFilterValidator(CustomValidator<ShippingQuery> validator) {
        super(ShippingQuery.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ShippingQueryFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.payments.ShippingQuery
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class ShippingQueryFilterValidator(validator: Validator<ShippingQuery>) : FilterValidator<ShippingQuery>(ShippingQuery::class, validator)
